interpreter/cling/lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -12,6 +12,10 @@
 #include "IncrementalExecutor.h"
 #include "cling/Utils/Platform.h"
 
+#ifdef __linux__
```

Comment:
```suggestion
#ifdef LLVM_ON_UNIX
```

Reply:
Perf is really Linux only, no other UNIX supported, it's literally part of the kernel. No point using some other define here.